src/d4bl/app/websocket_manager.py

```
@@ -22,7 +28,7 @@ async def send_websocket_update(job_id: str, message: dict) -> None:
     try:
         await websocket.send_json(message)
     except Exception as exc:  # noqa: BLE001 - best effort logging
-        print(f"Error sending WebSocket update: {exc}")
+        logger.error("Error sending WebSocket update: %s", exc)
```

Ruff (0.15.2)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)